x-pack/legacy/plugins/siem/public/components/load_more_table/index.tsx

```diff
@@ -199,101 +183,85 @@ export class LoadMoreTable<T, U, V, W, X, Y, Z, AA, AB> extends React.PureCompon
           {item.text}
         </EuiContextMenuItem>
       ));
+
     return (
-      <EuiPanel
+      <Panel
         data-test-subj={dataTestSubj}
+        loading={loading}
         onMouseEnter={this.mouseEnter}
         onMouseLeave={this.mouseLeave}
       >
-        <BasicTableContainer>
-          {loading && (
-            <>
-              <BackgroundRefetch />
-              <LoadingPanel
-                height="100%"
-                width="100%"
-                text={`${i18n.LOADING} ${loadingTitle ? loadingTitle : headerTitle}`}
-                position="absolute"
-                zIndex={3}
-                data-test-subj="LoadingPanelLoadMoreTable"
-              />
-            </>
-          )}
+        <HeaderPanel
+          id={id}
+          showInspect={!loadingInitial && this.state.showInspect}
+          subtitle={
+            !loadingInitial && `${i18n.SHOWING}: ${headerCount.toLocaleString()} ${headerUnit}`
+          }
+          title={headerTitle}
+          tooltip={headerTooltip}
+        >
+          {!loadingInitial && headerSupplement}
+        </HeaderPanel>
 
-          <HeaderPanel
-            id={id}
-            showInspect={this.state.showInspect}
-            subtitle={<>{`${i18n.SHOWING}: ${headerCount.toLocaleString()} ${headerUnit}`}</>}
-            title={headerTitle}
-            tooltip={headerTooltip}
-          >
-            {headerSupplement}
-          </HeaderPanel>
+        {loadingInitial ? (
+          <EuiLoadingContent data-test-subj="InitialLoadingPanelLoadMoreTable" lines={10} />
```

Comment:
nit: Your style usually looks to have `data-test-subj="` start with a lower case letter. Doesn't bother me though, but up to you if you want to keep them consistent.

Reply:
👍